.NET/Microsoft.Recognizers.Text.DataDrivenTests/Sequence/TestSequence_Chinese.cs

```diff
@@ -27,5 +27,12 @@ public void URLModel()
         {
             TestURL();
         }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "PhoneNumberModel-Chinese.csv", "PhoneNumberModel-Chinese#csv", DataAccessMethod.Sequential)]
+        [TestMethod]
+        public void PhoneNumberModel()
+        {
+            TestPhoneNumber();
```

Comment:
How does this method know to load the Chinese model and not the English one?

Reply:
> How does this method know to load the Chinese model and not the English one?

It calls Microsoft.Recognizers.Text.DataDrivenTests\TestContextExtensions.cs GetModelParseResults(), it will load the English Model.